tools/sdk-ai-bots/azure-sdk-qa-bot/test/thinkingHandler.test.ts

```diff
@@ -143,7 +143,7 @@ describe('ThinkingHandler', () => {
 
       const result = (thinkingHandler as any).generateAnswer(successResponse);
 
-      expect(result).toBe('This is a test answer without references');
+      expect(result).toContain('This is a test answer without references');
```

Comment:
In this way, it will not checkout one failed case which add extra content to the answer. The test case is not valid.
```suggestion
      expect(result).toBe('This is a test answer without references\n\n> **NOTE:** If you have follow-up questions after my response, please @Azure SDK Q&A Bot to continue the conversation.');
```

Reply:
Updated.